web/app_options.js

```diff
@@ -178,6 +178,11 @@ const defaultOptions = {
     value: typeof PDFJSDev === "undefined" || !PDFJSDev.test("CHROME"),
     kind: OptionKind.VIEWER + OptionKind.PREFERENCE,
   },
+  enableUpdatedAddImage: {
+    /** @type {boolean} */
```

Comment:
Is this intended to be a *temporary* option/pref?
If so, we should probably annotate it similar to the others:
```suggestion
  enableUpdatedAddImage: {
    // We'll probably want to make some experiments before enabling this
    // in Firefox release, but it has to be temporary.
    // TODO: remove it when unnecessary.
    /** @type {boolean} */
```